packages/components/src/Notification/ToastNotification/_docs/ToastNotification.stickersheet.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import { StickerSheetStory } from "~storybook/components/StickerSheet"
+import { addToastNotification } from "../subcomponents/ToastNotificationManager"
+
+export default {
+  title: "Components/Notifications/ToastNotification",
+  parameters: {
+    chromatic: { disable: false, delay: 600 },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: () => {
+    React.useEffect(() => {
+      addToastNotification({
```

Comment:
Hmm... this seems a little iffy. I was expecting a bunch of `<ToastNotification />`.
But just checked and it's how it was in legacy, so I suppose it's probably okay for now :s

Reply:
Yeah I'll leave it as is this round. I reckon some of the jank can be fixed refactoring the ToastNotificationManager